administrator-manual/en/tlspolicy.rst

```diff
@@ -17,6 +17,10 @@ The following sections describe each policy identifier.
 Policy 2018-03-30
 -----------------
 
+The goal of this policy is to harden the cipher set provided by the default
+upstream policy.  It is not compatible with IE 8 XP and Java 6u45 and 7u25
```

Comment:
There is a double space between "policy." and "It".

Reply:
I love to separate sentences with double space :blush: